### PR TITLE
Added alt text to two images

### PR DESCRIPTION
**Pull request by chrisdemars**
I added alternative text to the GitHub logo in the header and the `combineAll` graphic in the operators section.

### CODE REVIEW

**Comment by ladyleet** [comment]
@chrisdemars woohoo your first PR here! congrats!

**Comment by chrisdemars** [comment]
Stoked to get the first one in @ladyleet! 🔥

**Comment by ladyleet** [comment]
@chrisdemars small little conflicts with your branch - can you resolve?

**Comment by chrisdemars** [comment]
Sure @ladyleet

**Comment by chrisdemars** [comment]
I actually don't know which version is the correct one @ladyleet 🙁

**Comment by chrisdemars** [comment]
With help from @knittingcodemonkey, I think the merge is fixed.

**Comment by ladyleet** [comment]
great! need one more reviewer and someone to push! :) i've already approved.

File: src/app/operators/components/marble-diagram/marble-diagram.component.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <div class="marble-wrapper mat-elevation-z2">
   <img class="marble-diagram"
        [src]="url"
-       *ngIf="url" />
+       *ngIf="url"
+       alt="Diagram of how combineAll works" />
```
Comment (btroncone) [inline src/app/operators/components/marble-diagram/marble-diagram.component.html]: I think we need to make a small update so `combineAll` isn't hardcoded, something like:

`alt="Diagram of how {{operatorName}} works"`

Or we can add a getter to this component for the alt text:

```ts
get altText() {
  return `Diagram of how ${this.operatorName} works`;
}
```

then:

`[alt]="altText"`

**Comment by chrisdemars** [comment]
I believe I made the change that @btroncone suggested.

**Comment by chrisdemars** [comment]
I think this might be ready to be merged? Possibly?

**Comment by btroncone** [comment]
Ready to merge this but looks like TravisCI is blocking?

**Comment by benlesh** [comment]
Admin squash and merge because Travis was not reporting back at all and didn't appear to know about the build even.

**Comment by chrisdemars** [comment]
Thanks @benlesh and @knittingcodemonkey

**Comment by benlesh** [comment]
Thank YOU!

**Comment by ladyleet** [comment]
@chrisdemars srsly thank you so much for contributing! glad we finally got this in. :) and thank you @benlesh!

**Pull request merged**